Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -648,7 +648,18 @@ uint WrapCoord(int coord, uint wrap, int size) {{
               "  float3 coords = float3(float(uv.x) / size_s, float(uv.y) / size_t, layer);\n");
     if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
     {
-      out.Write("  return iround(255.0 * texture(tex, coords));\n}}\n");
+      if (!host_config.backend_sampler_lod_bias)
+      {
+        out.Write("  uint texmode0 = samp_texmode0(texmap);\n"
+                  "  float lod_bias = {} / 256.0f;\n",
+                  BitfieldExtract<&SamplerState::TM0::lod_bias>("texmode0"));
+      }
+      else
+      {
+        out.Write("  float lod_bias = 0.0f;\n");
+      }
+
+      out.Write("  return iround(255.0 * texture(tex, coords, lod_bias));\n}}\n");
```

Comment:
I feel like it would be cleaner to generate one of two different texture calls, one that has the bias and one that doesn't, instead of declaring `lod_bias` in two places.  There would still be the closing bracket, which needs to happen in both cases, but keeping a separate `Write` call for that after the if statement would work.